docs/source/example_use.rst

```diff
@@ -51,12 +51,31 @@ set up Orchestra yet, you should do that now. Also, make sure that
 have loaded the workflow into the database (``python manage.py loadworkflow
 journalism_workflow v1``).
 
-The journalism workflow requires Google Apps integration to run,
-so make sure in ``orchestra_settings.py`` you set ``settings.GOOGLE_APPS`` to
-``True``, and fill in values for ``settings.GOOGLE_SERVICE_EMAIL``,
-``settings.GOOGLE_P12_PATH``, and ``settings.GOOGLE_PROJECT_ROOT_ID``. Set up
-and correct values for those settings are described in `the Google Apps
-documentation <https://developers.google.com/drive/web/delegation>`_.
+The journalism workflow requires Google Apps integration to run, so
+make sure in ``orchestra_settings.py`` you set
+``settings.GOOGLE_APPS`` to ``True``, and fill in values for
+``settings.GOOGLE_SERVICE_EMAIL``, ``settings.GOOGLE_P12_PATH``, and
+``settings.GOOGLE_PROJECT_ROOT_ID``. Set up and correct values for
+those settings are described in `the Google Apps documentation
+<https://developers.google.com/drive/web/delegation>`_. Pay particular
+attention to the ``Authorizing requests with OAuth 2.0`` section. As a
+summary, the variables do the following:
+
+* ``settings.GOOGLE_SERVICE_EMAIL`` is the ID that Google will grant
+  you when you sign up for a `Google API console
+  account <https://console.developers.google.com/>`_.
```

Comment:
I haven't done this in a while, but is best practice to use the default service account? I thought you were supposed to go into the API credentials, create a new service account w/ a keypair, and download its .p12